Profiles - ZIB 2017/nl-core-contactpoint.xml

```diff
@@ -45,26 +45,29 @@
+        | Temporary Land Line/Tijdelijk telefoonnummer Vast telefoonnummer                      | LL               | phone   | temp   |&#xD;&#xA;&#xD;&#xA;
+        | Primary Work Land Line/Zakelijk telefoonnummer Vast telefoonnummer                    | LL               | phone   | work   |&#xD;&#xA;&#xD;&#xA;
+        |                                                                                       |                  |         |        |&#xD;&#xA;&#xD;&#xA;
+        | Primary Home Fax/Telefoonnummer thuis Fax                                             | FAX              | fax     | home   |&#xD;&#xA;&#xD;&#xA;
+        | Temporary Fax/Tijdelijk telefoonnummer Fax                                            | FAX              | fax     | temp   |&#xD;&#xA;&#xD;&#xA;
+        | Primary Work Fax/Zakelijk Telefoonnummer Fax                                          | FAX              | fax     | work   |&#xD;&#xA;&#xD;&#xA;
+        |                                                                                       |                  |         |        |&#xD;&#xA;&#xD;&#xA;
+        | Primary Home Mobile Phone/Telefoonnummer thuis Mobiel telefoonnummer                  | MC               | phone   | home   |&#xD;&#xA;&#xD;&#xA;
+        | Temporary Mobile Phone/Tijdelijk telefoonnummer Mobiel telefoonnummer                 | MC               | phone   | temp   |&#xD;&#xA;&#xD;&#xA;
+        | Primary Work Mobile Phone/Zakelijk telefoonnummer Vast telefoonnummer                 | MC               | phone   | work   |&#xD;&#xA;&#xD;&#xA;
+        |                                                                                       |                  |         |        |&#xD;&#xA;&#xD;&#xA;
+        | Primary Home Pager/Telefoonnummer thuis Pieper                                        | PG               | pager   | home   |&#xD;&#xA;&#xD;&#xA;
+        | Temporary Pager/Tijdelijk telefoonnummer Pieper                                       | PG               | pager   | temp   |&#xD;&#xA;&#xD;&#xA;
+        | Primary Work Pager/Zakelijk telefoonnummer Pieper                                     | PG               | pager   | work   |&#xD;&#xA;&#xD;&#xA;
+        |                                                                                       |                  |         |        |&#xD;&#xA;&#xD;&#xA;
+        | Private email address/Prive e-mailadres                                               |                  | email   | home   |&#xD;&#xA;&#xD;&#xA;
+        | Work email address/Zakelijk e-mailadres                                               |                  | email   | work   |&#xD;&#xA;&#xD;&#xA;
+        |                                                                                       |                  |         |        |&#xD;&#xA;&#xD;&#xA;
+        | Mobile business phone number/Zakelijk mobiel nummer                                   |                  | phone   | work   |&#xD;&#xA;&#xD;&#xA;

```

Comment:
Hoe is dit anders dan
`Primary Work Mobile Phone/Zakelijk telefoonnummer Mobiel telefoonnummer`
Waar ik hierboven al een comment over heb geplaatst (toevallig)?

Reply:
Zit wat in. Deze is nu zo als example opgenomen zonder zib-waardelijst-mapping, zodoende. Maar misschien is dat inderdaad wel overbodig. @ArdonToonstra wat denk jij?